Sources/CredentialsGitHub/CredentialsGitHub.swift

```diff
@@ -123,9 +126,12 @@ public class CredentialsGitHub : CredentialsPluginProtocol {
                                         try profileResponse.readAllData(into: &body)
                                         jsonBody = JSON(data: body)
 
-                                        if let id = jsonBody["id"].number?.stringValue {
-                                            let name = jsonBody["name"].stringValue
-                                            let userProfile = UserProfile(id: id, displayName: name, provider: self.name)
+                                        if let userProfile = self.createUserProfile(from: jsonBody) {
```

Comment:
Please convert jsonBody to a dictionary and change createUserProfile to accept a dictionary.
(We may be re-evaluating our use of SwiftyJSON in APIs).